sklearn/decomposition/pca.py

```diff
@@ -130,14 +130,19 @@ class PCA(_BasePCA):
 
             n_components == min(n_samples, n_features)
 
-        if n_components == 'mle' and svd_solver == 'full', Minka\'s MLE is used
-        to guess the dimension
-        if ``0 < n_components < 1`` and svd_solver == 'full', select the number
-        of components such that the amount of variance that needs to be
+        If ``n_components == 'mle'`` and ``svd_solver == 'full'``, Minka\'s
+        MLE is used to guess the dimension. ``n_components == 'mle'`` only
```

Comment:
the applicability only to full is stated on the previous line.

Maybe just say "use of n_components='mle'" will interpret "svd_solver='auto' as 'full'".